images-dist/modules/kogito-project-versions/module.yaml

```diff
@@ -18,19 +18,19 @@
 #
 schema_version: 1
 name: org.kie.kogito.project.versions
-version: "main"
+version: "1.37.0"
 description: "Kogito Project versions information"
 
 envs:
   - name: "KOGITO_VERSION"
-    value: "999-SNAPSHOT"
+    value: "9.104.0"
     description: Defines the Kogito version to be used by the builder images. Not intended to be changed by end user.
   - name: "QUARKUS_PLATFORM_GROUPID"
-    value: "io.quarkus.platform"
+    value: "com.redhat.quarkus.platform"
     description: Defines the Quarkus Platform groupId to be used by the builder images. Not intended to be changed by end user.
   - name: "QUARKUS_PLATFORM_VERSION"
     value: "3.20.1"
```

Comment:
Just a comment, since you probalby align this at a a later point.
But shouldn't this quarkus version be an RHBQ -> 3.20.1.redhat-00003 instead?

Reply:
But the diff is strange, the actual HEAD of `9.104.x-prod` is different